app/build-js.sh

```diff
@@ -24,7 +24,7 @@ fi
 pushd ../node_modules/astro-sdk
 npm install
 grunt build_astro_client
-cp js/build/astro-client.js $MYPATH/scaffold-www/
```

Comment:
do we need to make a change like this in matalan? because the `js` folder is also new there